packages/cli/src/index.ts

```diff
@@ -17,19 +17,11 @@ program
 program
   .command('dev')
   .description('Start a Novu Dev Studio and a localtunnel')
-  .option('-p, --port <value>', 'Set the local port for the Novu endpoint, defaults to 4000', '4000')
-  .option('-r, --route <value>', 'Set the Novu Endpoint mounted route, defaults to /api/novu', '/api/novu')
-  .option('-o, --origin <value>', 'Set the origin for the Novu endpoint')
-  .option('-sp, --studio-port <value>', 'Set the local port for the Novu Local Studio server, defaults to 2022', '2022')
-  .option(
-    '-so, --studio-remote-origin <value>',
-    'Set the remote origin for Novu Studio, used for staging environment and local development, defaults to https://web.novu.co',
-    'https://web.novu.co'
-  )
-  .option(
-    '--region <value>',
-    'Studio Origin SPA, used for staging environment and local development, defaults to us',
-    'us'
-  )
+  .option('-p, --port <value>', 'The port for the local Bridge endpoint', '4000')
+  .option('-r, --route <value>', 'The Bridge endpoint route', '/api/novu')
+  .option('-o, --origin <value>', 'The origin for the Bridge endpoint')
+  .option('-r, --region <value>', 'Novu Cloud region, for staging and local development', 'us')
+  .option('-wu, --web-url <value>', 'The url for Novu Web, for staging and local development', 'https://web.novu.co')
+  .option('-wp, --web-port <value>', 'The port for the Novu Studio server', '2022')
```

Comment:
I'm not sure about this one, it's not really the web port IMO. 

It's a --studio-port, since the web port is always 443. 

I think there should be a different between web and studio:
- web: https://web.novu.co
- studio: localhost:2022

wdyt?

Reply:
Good point. I'll update.